Update translation of strings to Spanish (GUI) #393

huguei opened the pull request:
## Purpose

Update of spanish translation messages.es.xlf file.

## Context

Fixes #393

## Changes

src/locale/messages.es.xlf

matsduf commented:
Thank you, @huguei!

In the table there are two "Undelegated" which you both translated to "No está delegado", which is fine if Spanish does not make a distinction between singular and plural context. English does not, but other languages like French and Swedish does. The first is for plural context and the second for singular context, which is noted in the notes for the translators which is shown in the lower right corner of the Poedit window (if you use that).

That was the only thing I saw, but I do not read Spanish.

huguei commented:
> Thank you, @huguei!
> 
> In the table there are two "Undelegated" which you both translated to "No está delegado", which is fine if Spanish does not make a distinction between singular and plural context. English does not, but other languages like French and Swedish does. The first is for plural context and the second for singular context, which is noted in the notes for the translators which is shown in the lower right corner of the Poedit window (if you use that).
> 
> That was the only thing I saw, but I do not read Spanish.

Sorry about that, I didn't notice the translator notes in poedit!
I've just reviewed again all the notes, and corrected this and other message. Also deleted an extra dot at the end of a message.
Thanks.

matsduf commented:
> Sorry about that, I didn't notice the translator notes in poedit! I've just reviewed again all the notes, and corrected this and other message. Also deleted an extra dot at the end of a message. Thanks.

Thank you! I think it is easy to miss the translator notes.
